driver/e2e_test.go

```diff
@@ -32,7 +33,7 @@ func TestQuery(t *testing.T) {
 		{"Select Name", "SELECT name FROM db.person", nil, []any{&name}, records.Columns(1)},
 		{"Select Count", "SELECT COUNT(1) FROM db.person", nil, []any{&count}, Records{{len(records)}}},
 
-		{"Insert", `INSERT INTO db.person (name, email, phone_numbers, created_at) VALUES ('foo', 'bar', '["baz"]', NOW())`, nil, []any{}, Records{}},
+		{"Insert", `INSERT INTO db.person (name, email, phone_numbers, created_at) VALUES ('foo', 'bar', '["baz"]', ?)`, []any{now}, []any{}, Records{}},
```

Comment:
can we add a new test rather than replacing an existing one?

Reply:
Thank you for reviewing.
I have incorporated your suggestion.